src/types/desktop/maintenanceTypes.ts

```diff
@@ -0,0 +1,66 @@
+import type { Ref } from 'vue'
```

Comment:
We can remove all readonly from the field in this file and mark the the declaration in `/constants` const.